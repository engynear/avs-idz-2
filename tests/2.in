#include <stdio.h>#include <string.h>int main() {char str[1000];int i, if_count = 0, do_count = 0, int_count = 0, for_count = 0, else_count = 0;fgets(str, 1000, stdin);for (i = 0; i < strlen(str); i++) {if (str[i] == 'i' && str[i + 1] == 'f') {