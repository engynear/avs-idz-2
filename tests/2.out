if: 2, do: 1, count: 3, for: 2, else: 1